### ISSUE TITLE
Create single `config.json` for both relayers and UI app

### ISSUE THREAD

**Issue by isSerge** [subspace/subspace-relayer#130]
Currently, we have three configs:
- `config.json` for Kusama and parachains
- `config.json` for Polkadot and parachains
- The UI app config in `config/AvailableParachain.ts`

Problems:
- Every time we add a new parachain we have to manually update two of these
- Due to feed ownership we have to maintain `feedId` sequence within each `config.json`. Otherwise, one account will create a feed with a different `feedId` and won't be able to submit blocks in the original feed

Possible solution: create a single `config.json` that will have all chains and be used by both relayers and the UI app. This way we only need to do updates in one place. Feeds creation script can be updated to create all feeds in one go (no need to manually update `feedId` for each chain every time we deploy a new relaynet)

**Comment by nazar-pc** [comment]
This only works if everything is run by the same entity (us). As long as UI needs to show feeds submitted by someone else (ideally there would be per-feed keypair that isn't derived from the same seed) this approach will stop working.

**Comment by isSerge** [comment]
UI requires one entity anyway - someone has to add chain logo and other data to config

**Comment by nazar-pc** [comment]
I mean we have Polkadot.js apps as an example where it is technically controlled by Parity, but it doesn't mean that they also controls all the RPC endpoints of all chains in there.

I guess it boils down to how generic and decoupled we want frontend and backend to be.

**Comment by isSerge** [comment]
It can work the same way: anybody can create a feed and submit data, but in order to be displayed on relayer UI - one has to submit PR

**Comment by nazar-pc** [comment]
Right, but if the config is one for everything, how would that work?

**Comment by isSerge** [comment]
We can start with one config, in case we have independent feeds, we can create a separate config for UI only, which can be updated separately and also generated by script based on the changes in the relayer config. Currently I just want to minimize manual work to update configs and reduce possible errors

**Comment by nazar-pc** [comment]
I don't have strong opinion here, feel free to send a PR to show how it works
